x-pack/plugins/ml/server/lib/sample_data_sets/sample_data_sets.ts

```diff
@@ -18,13 +18,6 @@ export function addLinksToSampleDatasets(server: any) {
     icon: 'machineLearningApp',
   });
 
-  server.addAppLinksToSampleDataset('flights', {
-    path:
-      '/app/ml#/modules/check_view_or_create?id=sample_data_flights&index=d3d7af60-4c81-11e8-b3d7-01146121b73d',
-    label: sampleDataLinkLabel,
-    icon: 'machineLearningApp',
-  });
-
   server.addAppLinksToSampleDataset('logs', {
     path:
       '/app/ml#/modules/check_view_or_create?id=sample_data_weblogs&index=edf84fe0-e1a0-11e7-b6d5-4dc382ef7f5b',
```

Comment:
The ID of the index pattern was incorrect. Can you edit it to `90943e30-9a47-11e8-b64d-95841ca0b247`